### DISCUSSION TITLE
Can I localise an element in the top bar?

### DISCUSSION THREAD

**Discussion by its-miroma** [vuejs/vitepress#3922] [Q&A] [answered]
The Fabric documentation website contains a link to a `contributing` page in the top bar, which [currently] links to `/contributing`. That is an absolute link, which means that all locales (for example `/it_it`) will redirect to English, and not to the localised version (for example `/it_it/contributing`).

[I tried] changing the link to `./contributing` (relative), but that only works in root (for example in `/it_it/develop` the link opens `/it_it/develop/contributing`, but I want it to open `/it_it/contributing`).

What can I do?

---

A possible hacky solution would be the following redirects:

- `/la_ng/**/contributing` → `/la_ng/contributing`
- `/**/contributing` → `/contributing`

...but I do not think Vitepress supports redirects. I don't think `rewrites` works because the files do not exist.

[currently]: <https://github.com/FabricMC/fabric-docs/blob/2ed8096a2476713988c798d3fd7943ce8b7f6aed/.vitepress/config.mts#L83>
[I tried]: <https://github.com/FabricMC/fabric-docs/pull/100>

**Comment by brc-dd** [comment]
You can just specify different url for different locale 👀

**Reply by its-miroma** [reply]
So I must create a different `ThemeConfig` for each locale?

**Reply by brc-dd** [reply, marked as answer]
Yeah. Store it in a separate variable and merge that inside `loadLocales` or something. You can also refactor your `generateTranslatedSidebars` function to merge/override sidebar directly in loadLocales.
